vllm/model_executor/layers/quantization/gptq_marlin.py

```diff
@@ -62,7 +65,14 @@ def _get_perms():
     return perm, scale_perm, scale_perm_single
 
 
-_perm, _scale_perm, _scale_perm_single = _get_perms()
+_perm = {}
```

Comment:
I wonder why you need these global variables -- if you want to avoid recomputing things, it is most likely cleaner / better to use a function with a `@functools.lru_cache` annotation :)

Reply:
I just realized that we don't need most of this code anymore since we use auto-repack GPU kernel. Refactored to use only scale shuffles.